src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -281,6 +281,14 @@
     <value>Automatically detect URLs and make them clickable</value>
     <comment>Header for a control to toggle whether the terminal should automatically detect URLs and make them clickable, or not.</comment>
   </data>
+  <data name="Globals_SearchWebDefaultQueryUrl.Header" xml:space="preserve">
+    <value>Query URL for "Search web" action</value>

```

Comment:
I think we should avoid using the term "query" as it's the `?part` at the end of an URL. Maybe `URL used for the "Search web" action`.